Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -265,6 +265,46 @@ class StoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
         expect(eligibility) == .eligible
     }
 
+    func testIneligibleForIntroForDifferentProductInSameSubscriptionGroupAfterPurchase() async throws {
+        try XCTSkipIf(
+            Self.storeKit2Setting == .enabledForCompatibleDevices,
+            "This test currently does not pass with SK2 (see FB11889732)"
```

Comment:
is this one under the company account? I can't find it

Reply:
It is. Click on "Inbox" instead of "Submitted" because it got a response so it's there.

Reply:
Thanks!! That's exactly what happened 🤦 I was only checking "submitted"